frontend/src/components/user/CategoryCard.tsx

```diff
@@ -7,45 +9,71 @@ import LessonConfirmationDialog from './LessonConfirmationDialog'
 type Props = {
   category: ICategory
   author: IUser
+  delay: number
 }
 
-const CategoryCard: React.FC<Props> = ({ category, author }): JSX.Element => {
+const CategoryCard: React.FC<Props> = ({ category, author, delay }): JSX.Element => {
   const [isOpen, setIsOpen] = useState(false)
 
-  const closeModal = () => setIsOpen(false)
-  const openModal = () => setIsOpen(true)
+  const toggle = () => setIsOpen(!isOpen)
 
   const get_author_lessons = category?.lessons?.map(({ user_id }: ILesson) => user_id === author?.id)
 
   const is_already_taken: any = get_author_lessons?.filter(value => value)
 
   return (
-    <div className={classNames(
-      'bg-white rounded-lg shadow-sm border hover:shadow-xl',
-      'transition ease-in-out duration-200'
-    )}>
-      <div className="flex flex-col justify-between px-5 py-5">
-        <div className="space-y-3 min-h-[160px]">
-          <h5 className="text-lg font-semibold tracking-tight">{category?.title}</h5>
-          <p className="text-sm line-clamp-6">{category?.description}</p>
-        </div>
-        <div className="mt-3 flex justify-end">
-          <button 
-            type="button"
-            disabled={is_already_taken[0]}
-            onClick={openModal}
-            className="btn-primary"
-          >
-            {is_already_taken[0] ? 'Done' : 'Start'}
-          </button>
-          <LessonConfirmationDialog 
-            isOpen={isOpen}
-            closeModal={closeModal}
-            category_id={category?.id}
-          />
+    <section className="section-sm">
+      <div className="flex flex-col lg:flex-row lg:gap-x-[33px] gap-y-24">
+        <div
+          className={classNames(
+            'w-full max-w-[368px] px-[18px] pb-[26px] lg:px-[28px] lg:pb-[38px] bg-white',
+            'hover:shadow-primary flex flex-col rounded-[14px] mx-auto transition'
+          )}
+          data-aos="fade-up"
+          data-aos-delay={`${400+(delay+200)}`}
+        >
+          <div className="-mt-[38px] lg:-mt-12 mb-4 lg:mb-6">
+            <Image 
+              src="/img/category.jpg" 
+              width={312}
+              height={268}
+              className="rounded-2xl" 
+              alt="category"
+              blurDataURL="/img/category.jpg"
+              placeholder="blur"
+              quality={100} 
+              layout="intrinsic"
+            />
+          </div>
+          <div>
+            <h4 className="text-lg lg:text-xl font-semibold mb-2 lg:mb-4">
+              {category.title}
+            </h4>
+            <p className="line-clamp-4">{category.description}</p>
+          </div>
+          <div className="flex items-center justify-between mt-8 mb-2 lg:mb-0">
+            {/* stars */}
+            <div className="flex gap-x-2 text-red-400">
+              {[0,1,2,3,4].map((i) => <BsStarFill key={i} />)}
```

Comment:
What are `01234` values for? Are these constants? If so place them inside a constant variable

Reply:
THat is for the number of stars